hledger-lib/Hledger/Reports/PostingsReport.hs

```diff
@@ -38,27 +38,25 @@ import Hledger.Reports.ReportOptions
 -- transaction info to help with rendering.
 -- This is used eg for the register command.
 type PostingsReport = [PostingsReportItem] -- line items, one per posting
-type PostingsReportItem = (Maybe Day    -- The posting date, if this is the first posting in a
-                                        -- transaction or if it's different from the previous
-                                        -- posting's date. Or if this a summary posting, the
-                                        -- report interval's start date if this is the first
-                                        -- summary posting in the interval.
-                          ,Maybe Day    -- If this is a summary posting, the report interval's
-                                        -- end date if this is the first summary posting in
-                                        -- the interval.
-                          ,Maybe Text   -- The posting's transaction's description, if this is the first posting in the transaction.
-                          ,Posting      -- The posting, possibly with the account name depth-clipped.
-                          ,MixedAmount  -- The running total after this posting, or with --average,
-                                        -- the running average posting amount. With --historical,
-                                        -- postings before the report start date are included in
-                                        -- the running total/average.
+type PostingsReportItem = (Maybe Day     -- The posting date, if this is the first posting in a
+                                         -- transaction or if it's different from the previous
+                                         -- posting's date. Or if this a summary posting, the
+                                         -- report interval's start date if this is the first
+                                         -- summary posting in the interval.
+                          ,Maybe Period  -- If this is a summary posting, the report interval's period.
+                          ,Maybe Text    -- The posting's transaction's description, if this is the first posting in the transaction.
+                          ,Posting       -- The posting, possibly with the account name depth-clipped.
+                          ,MixedAmount   -- The running total after this posting, or with --average,
+                                         -- the running average posting amount. With --historical,
+                                         -- postings before the report start date are included in
+                                         -- the running total/average.
                           )
 
 -- | A summary posting summarises the activity in one account within a report
 -- interval. It is kludgily represented by a regular Posting with no description,
 -- the interval's start date stored as the posting date, and the interval's end
 -- date attached with a tuple.
```

Comment:
My comment here looks a bit sketchy.. and out of date now, hopefully ?

Reply:
Sorry, I may have misclicked. I'm wondering if the "It is kludgily represented..." haddock needs updating.

Reply:
You are correct. I've updated it.